MediaBrowser.Plugins.Anime/Providers/MyAnimeList/api.cs

```diff
@@ -28,16 +26,15 @@ public class Api
         public string anime_link = "https://myanimelist.net/anime/";
 
 
+        public Api()
```

Comment:
Maybe I'm confusing languages, but the default constructor doesn't need to be explicitly defined, right?

Reply:
That's true, I'll remove it

Reply:
since I have force pushed that commit is gone, so the original change here is that logManager is gone from the constructor. I left the empty constructor because I was uncertain if it had to be there